scripts/build-image.sh

```diff
@@ -111,6 +111,11 @@ find /var/log -type f -print0 | xargs -0r rm -rf
 find /var/lib/apt/lists -type f -print0 | xargs -0r rm -rf
 find /usr/local/lib/python2.7/dist-packages/github/tests -type f -print0 | xargs -0r rm -rf
 
+# Configure the path for the postgres logs
+mkdir -p /var/log/pg_log
+chmod 0757 /var/log/pg_log/
+echo -e "export PG_LOG_PATH=/var/log/pg_log\n" | tee -a /etc/bash.bashrc
```

Comment:
@ruiztulio This is not the current pg logs directory.

Thery are in `'/var/lib/postgresql/%s/main/pg_log/postgresql.log' % pg_version`

Somehow it is not taken the path that you are setting

Reply:
@JesusZapata  Maybe we should change the shippable image in order to use the path: `/var/log/pg_log` because @ruiztulio told me that is the default one.

Note: A `chown` is required in order to allow read this log for `odoo` OS user.

Reply:
ACK

Reply:
@JesusZapata @hugho-ad 
We will have issues if there are 2 postgresql up and they are using the same file...

Let me discuss that with @ruiztulio before change it

For now @JesusZapata help us to overwrite this environment variable from shippable image

Reply:
@moylop260 yes, that is the default AFAIK.

@JesusZapata @moylop260 I'm not worried about multiple Postgres instances, because that should not happend, if you are going to profile Odoo with 2 postgres versions inside a container you do it with one then the other not both at the same time in the same container.

In the other hand the test instances that will have the profiler module enables will only have one postres version available, the one used in production and if for some reason another version is required it won't be for the same instance.

Perhaps I'm missing something.